src/material-experimental/mdc-helpers/_mdc-helpers.scss

```diff
@@ -51,9 +52,45 @@ $mat-typography-level-mappings: (
   )
 );
 
+// Mappings between Angular Material and MDC's typography levels.
```

Comment:
Could you expand this comment to be explicit about the direction of the mapping? I.e.
```scss
// Maps ___ to ____. 
```
Might also be good to touch here on how this is intended to be used